app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html

```diff
@@ -65,7 +68,10 @@ <h1 class="govuk-heading-l">@messages("dashboard.h1")</h1>
 @unavailableContent = {
     <h1 class="govuk-heading-l">@messages("dashboard.h1")</h1>
     <p class="govuk-body">@Html(messages("dashboard.p1", s"<b>${Formatters.instantFormatHours(Instant.now)}</b>"))</p>
-    <strong class="@statusClass">@messages("service.state.UNAVAILABLE")</strong>
+    <label class="govuk-label govuk-visually-hidden" for="service-status">
+        @messages("dashboard.status.screen.reader", serviceId)
```

Comment:
Won't this just result in the resolved message being: 

"Availability status for the haulier"

?

If doing it this way, instead of just supplying the serviceId, doesn't the arg passed to messages need to itself be a message based on the serviceId which resolves to:

"Goods Vehicle Movement Service"

and doesn't it also need to include the status string (Available or Unavailable accordingly).

Reply:
In the prototype, it looks like this: <img width="546" alt="Screenshot 2024-02-26 at 10 18 10" src="https://github.com/hmrc/customs-service-status-frontend/assets/23785/4a1e51f8-5b3d-4348-a5d6-fd069880d981">

The status is separated from the hidden message. 

Thus my code is like this: https://github.com/hmrc/customs-service-status-frontend/blob/b065079561ed71dd09e548c91348c4a55d8ab9e9/app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html#L71-L74

Reply:
Now, update it with service name.

Reply:
We can't do it by retrieving the "name" from the backend DB, as that field is going to be removed (discussed in refinement). This is display content which will be handled in messages. That's why in the comment above I mentioned that we need a message key based on the service id whose value is the message to use. 

So, if you're going to do it the way you currently are, you'd presumably need to adapt this:

@messages("dashboard.status.screen.reader", serviceId) 

to something like this: 

@messages("dashboard.status.screen.reader", s"dashboard.status.$serviceId.longName") 

and then in messages have: 

dashboard.status.haulier.longName=Goods Vehicle Movement Service

Understand what you're saying about the status string being in a separate element in prototype.